lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
Minor: Thinking about it a bit more, rather than use `Hash#dig`, perhaps we could extract a method like this to avoid the `if` statement:

```ruby
def ceremony_country_data
  @services_data[ceremony_country] || {}
end

def services_payment_partial_name
  ceremony_country_data['payment_partial_name']
end
```